site/content/user-guide/http-adapter.md

```diff
@@ -22,10 +22,12 @@ There is a subtle difference between the *device identifier* (*device-id*) and t
   * (required) `Authorization`: The device's *auth-id* and plain text password encoded according to the [Basic HTTP authentication scheme](https://tools.ietf.org/html/rfc7617).
   * (required) `Content-Type`: The type of payload contained in the body.
   * (optional) `hono-ttd`: The number of seconds the device will wait for the response.
+  * (optional) `QoS-Level`: The QoS level for publishing telemetry messages. Only QoS 1 is supported by the adapter.
```

Comment:
It would be nice if you could add an example for using this header in the Examples section below.